Source/Virtusize.swift

```diff
@@ -24,20 +24,42 @@
 
 import Foundation
 
+/// The main class used by Virtusize clients to perform all available operations related to fit check
 public class Virtusize {
+    // MARK: - Properties
+
+    /// The API key that is unique and provided for Virtusize clients
 	public static var APIKey: String?
+
+    /// The user id that is the unique user id from the client system
 	public static var userID: String?
+
+    /// The Virtusize environment that defaults to the `global` domain
 	public static var environment = VirtusizeEnvironment.global
-    // Default to the language set fot the phone
+
+    /// The device language that defaults to the primary device language
     public static var language: String = Locale.preferredLanguages[0]
-    // `Virtusize.productDataCheckDidFail`, the `UserInfo` will contain a message
-    // with the cause of the failure
+
+    /// NotificationCenter ob for debugging the initial product data check
```

Comment:
Is the `ob` here corerect?

Reply:
I was meant to type observers, but somehow I didn't finish the typing.